frontend/integration-tests/tests/cluster-settings.scenario.ts

```diff
@@ -0,0 +1,126 @@
+    await crudView.isLoaded();
+    expect(clusterSettingsView.channelUpdateLink.isDisplayed()).toBe(true);
+    await clusterSettingsView.channelUpdateLink.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.channelDropdownButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.channelDropdownButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.getSelectedChannel.get(0).isDisplayed()).toBe(true);
+    await clusterSettingsView.getSelectedChannel.get(0).click();
+
+    expect(clusterSettingsView.channelPopupCancelButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.channelPopupCancelButton.click();
+    await crudView.isLoaded();
+  });
+  it('display Cluster Operators page, click resource item link to display details. Check if the resource link title equals the details page header', async()=>{
+
+    await horizontalnavView.clickHorizontalTab('Cluster Operators');
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.clusterOperatorResourceLink.isDisplayed()).toBe(true);
+    await clusterSettingsView.clusterOperatorResourceLink.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.clusterResourceDetailsTitle.isDisplayed()).toBe(true);
+    expect(clusterSettingsView.clusterResourceDetailsTitle.getText()).toBe('console');
+
+    await horizontalnavView.clickHorizontalTab('YAML');
+    await crudView.isLoaded();
+
+  });
+  it('display Global Configuration page, click Configuration Resource item link and display details. Check if the resource link title equals the details page header', async() => {
+
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.globalConfigResourceLink.isDisplayed()).toBe(true);
+    await clusterSettingsView.globalConfigResourceLink.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.clusterResourceDetailsTitle.isDisplayed()).toBe(true);
+    expect(clusterSettingsView.clusterResourceDetailsTitle.getText()).toBe('cluster');
+
+    await horizontalnavView.clickHorizontalTab('YAML');
+    await crudView.isLoaded();
+
+  });
+  it('display Global Configuration page, click dropdown link to edit resource and display details, and check if details header is correct.', async() => {
+
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+
+    await clusterSettingsView.globalConfigResourceRow.click();
+    await crudView.isLoaded();
+
+    expect(crudView.actionForLabel('Edit Console Resource').isDisplayed()).toBe(true);
+    await crudView.actionForLabel('Edit Console Resource').click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.clusterResourceDetailsTitle.isDisplayed()).toBe(true);
+    expect(clusterSettingsView.clusterResourceDetailsTitle.getText()).toBe('cluster');
+
+    await horizontalnavView.clickHorizontalTab('Overview');
```

Comment:
I would skip the tests from here until the end of this function. They're generic details page actions that are already covered by the existing crud scenario.

Reply:
Make sure to rollback other changes like new `data-test-id`s if we remove some checks and they're no longer needed.

Reply:
Will remove them accordingly. Left them for View Schema integration tests which would be another PR, and thought of making use of them soon.